coalib/settings/FunctionMetadata.py

```diff
@@ -2,20 +2,22 @@
 from copy import copy
 from inspect import getfullargspec, ismethod
 
+from coalib.misc.Decorators import enforce_signature
 from coalib.settings.DocumentationComment import DocumentationComment
 
 
 class FunctionMetadata:
     str_nodesc = "No description given."
     str_optional = "Optional, defaults to '{}'."
 
+    @enforce_signature
```

Comment:
I just want an exception rather than getting something like "desc is none" later, this just gives us some safety